Server/src/index.ts

```diff
@@ -127,6 +128,12 @@ export class CitrineOSServer {
     this._server =
       server || fastify().withTypeProvider<JsonSchemaToTsProvider>();
 
+    // enable cors
+    this._server.register(cors, {
+      origin: true, // This can be customized to specify allowed origins
+      methods: ['GET', 'POST', 'PUT', 'DELETE'], // Specify allowed HTTP methods
```

Comment:
no PATCH? (if that's even a valid method in this case)

Reply:
I don't think we have any PATCHes right now